wazo_sysconfd/plugins/asterisk/asterisk.py

```diff
@@ -5,25 +5,22 @@
 import shutil
 import subprocess
 
-from xivo import http_json_server
-from xivo.http_json_server import HttpReqError
-from xivo.http_json_server import CMD_R
+from wazo_sysconfd.exceptions import HttpReqError
 
 ASTERISK_USER = 'asterisk'
 ASTERISK_GROUP = 'asterisk'
 
 
 class Asterisk(object):
-
     def __init__(self, base_vmail_path='/var/spool/asterisk/voicemail'):
         self._base_vmail_path = base_vmail_path
         self.remove_directory = _remove_directory
         self.move_directory = _move_directory
         self.is_valid_path_component = _is_valid_path_component
 
     def delete_voicemail(self, args, options):
-        if 'mailbox' not in options:
-            raise HttpReqError(400, "missing 'mailbox' arg", json=True)
+        if not options['mailbox']:
```

Comment:
`if not options['mailbox']` is not the same as `if 'mailbox' not in options`. One tests if a key is present in the dictionary, one tests if the value is False-y.